ipvs.go

```diff
@@ -0,0 +1,199 @@
+package procfs
+
+import (
+	"bufio"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net"
+	"strconv"
+	"strings"
+)
+
+type IPVSStats struct {
```

Comment:
Something I'm really not sure about is naming here: stats vs. stat vs status vs statuses?